packages/linter/src/rules/NoIconFontIsUsed.ts

```diff
@@ -41,6 +42,28 @@ export class NoIconFontIsUsed extends Rule {
       return this.pass();
     }
 
+    // check for known external stylesheets
+    const knownExternalStylesheets = ICON_FONT_IDENTIFIERS.filter(
+      (identifier) => {
+        if (identifier.hasOwnProperty("url") && identifier["url"]) {
+          return (
+            $(`link[rel='stylesheet'][href^='${identifier.url}']`).length > 0
+          );
+        }
+      }
+    );
+
+    if (knownExternalStylesheets.length) {
+      return this.fail("It seems like icon fonts are being used on this page.");
```

Comment:
Rephrase to 'Avoid using icon fonts to improve loading speed and accessibility'.